app/src/main/java/com/yacgroup/yacguide/database/tourbook/JsonExporter.kt

```diff
@@ -17,50 +17,39 @@
 
 package com.yacgroup.yacguide.database.tourbook
 
-import android.content.ContentResolver
 import android.net.Uri
 import com.yacgroup.yacguide.database.Ascend
-import com.yacgroup.yacguide.database.DatabaseWrapper
 import org.json.JSONArray
 import org.json.JSONException
 import org.json.JSONObject
 import java.io.IOException
 
-class JsonExporter(
-    private val _db: DatabaseWrapper,
-    _contentResolver: ContentResolver): BaseExporter(_contentResolver) {
-
-    private val JSON_INDENT_SPACE = 2
+open class JsonExporter: BaseExporter() {
+    companion object {
+        const val JSON_INDENT_SPACE = 2
```

Comment:
At this point, I'm a bit unsure. What I would like to have, is a constant that is also visible in the child class and not visible from outside. I thought `protected` does this but defining a `protected val` in uppercase characters, leads to a naming "issue". The IDE says, that properties shall be start with a lowercase character. A `protected const val` inside `companion object` cannot be accessed from the child.

Reply:
OK. This seems to be a much better solution. Thanks,